src/main/java/io/aiven/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -213,6 +233,46 @@
                 ConfigDef.Width.LONG,
                 TABLE_NAME_FORMAT_DISPLAY
             )
+            .define(
+                TABLE_NAME_NORMALIZE,
+                ConfigDef.Type.BOOLEAN,
+                TABLE_NAME_NORMALIZE_DEFAULT,
+                ConfigDef.Importance.MEDIUM,
+                TABLE_NAME_NORMALIZE_DOC,
+                DATAMAPPING_GROUP,
+                2,
+                ConfigDef.Width.LONG,
+                TABLE_NAME_NORMALIZE_DISPLAY
+            )
+            .define(
+                TOPICS_TO_TABLES_MAPPING,
+                ConfigDef.Type.STRING,
```

Comment:
There's `ConfigDef.Type.LIST`, would you consider using it?

Reply:
true !!!!